app.py

```diff
@@ -215,11 +212,12 @@ def create_folder_section(folder_number, section_name, db_get):
     section_object = db_get(folder_number)
     action = "add"
     status = "To be filled"
+    last_modified_on = datetime.datetime.today().strftime('%Y-%m-%d')
     if section_object is not None:
         action = "edit"
-        status = ""
-    section = FolderSection(section_name, action, status,  last_modified_by="Who Knows",
-                            last_modified_on=datetime.datetime.today().strftime('%Y-%m-%d'))
+        status = section_object.fld_form_status.data
+        last_modified_on = section_object.last_update.data
+    section = FolderSection(section_name, action, status, last_modified_on = last_modified_on, last_modified_by="Who Knows", )
```

Comment:
What is FolderSection?

Reply:
no idea 😀 I actually thought that was your code but it seems to be somethng I messsed with.. again may have been part of attempt to get username